enterprise/metrics/src/test/java/org/neo4j/metrics/MetricsKernelExtensionFactoryIT.java

```diff
@@ -88,6 +89,7 @@ public void mustLoadMetricsExtensionWhenConfigured() throws Exception
                 tx.success();
             }
         }
+        cluster.stop();
```

Comment:
At this point the cluster is not needed. The rest of that test is only looking at the output log file. It's good to cleanup your mess closer to the source.